package.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>urdf_geometry_parser</name>
+  <version>0.0.0</version>
```

Comment:
It was a request of @bmagyar to reset the version number to 0.0.0 https://github.com/ros-controls/urdf_geometry_parser/pull/2#discussion_r117959144
The version number will be incremented when the package is released right ?